CRM/CiviAwards/Setup/ProcessAwardsCategoryForCustomGroupSupport.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use CRM_CiviAwards_Helper_CaseTypeCategory as CaseTypeCategoryHelper;
+use CRM_Civicase_Service_CaseCategoryCustomDataType as CaseCategoryCustomDataType;
+use CRM_Civicase_Service_CaseCategoryCustomFieldExtends as CaseCategoryCustomFieldExtends;
+
+/**
+ * Class CRM_CiviAwards_Setup_ProcessAwardsCategoryForCustomGroupSupport.
+ */
+class CRM_CiviAwards_Setup_ProcessAwardsCategoryForCustomGroupSupport {
+
+  const AWARDS_CATEGORY_CG_LABEL = 'Case (Awards)';
```

Comment:
Do we need to localise Label?

Reply:
Not needed @erawat , Civi does that already in forms where the options are displayed. This value is only for inserting in DB